docs/data/sources/google-tag-manager-server.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Amplitude Analytics
+description: Official Google Tag Manager template for Amplitude Analytics for data collection.
```

Comment:
Please be more explicit about which template this is.

Reply:
Changed to the following:

Collect data with ease using Amplitude Analytics GTM template - the official server-side Google Tag Manager template for seamless data collection